src/store/hdap.js

```diff
@@ -0,0 +1,396 @@
+import { defineStore } from 'pinia'
+import { ref } from 'vue'
+
+async function action(apiBase, expirationDate, jwt, name, id, params, body = {}) {
```

Comment:
These could be simpler if they were inside `useHdapStore`.

I guess it's okay to have "private" functions in there. Just don't return them?

Reply:
Fixed in my copy